### ISSUE TITLE
Echelon CX5 + QZ App + FulGaz on PC

### ISSUE THREAD

**Issue by robtheviking** [cagnulein/qdomyos-zwift#1378]
Hi Roberto,
I am using my Echelon CX5 with FulGaz and tested out QZ with it to see if it will work with FulGaz "Reactive" mode. Basically, realistic riding. Here is the debug log (please change the file extension to .zip in case there is some issue with downloading a .zip)

https://mega.nz/file/jRRnWboR#jCEXJrMbckk7P0zMmKrqgryhePFViszkdZZKqN_MSYQ

I am seeing that even though my echelon stops pedaling (and the rpm is seen as 0, the fulgaz app still proceeds, is this an issue with QZ or fulgaz?)

The original issue was that the power reading was fluctuating randomly like it was all noise. in this run, that doesnt seem to be the case. Maybe it was a ulucky when I tried it before but I guess but that time I was using my VR headset with Windows Desktop. 

Fulgaz results video with 0rpm:
https://mega.nz/file/mE40mS6K#heovFU9L5veUIPhsUZjRNoir8wikN80SQtary_I8RfM

The FulGaz settings:

Settings->Performance Settings->Advanced->Aerodynamic Drag = 0.33
Settings->Performance Settings->Advanced->Rolling Resistance = 0.0033
Settings->Performance Settings->Advanced->Wind = OFF

Settings->Trainer Settings-> Real Road Feel = OFF
Settings->Trainer Settings->Advanced Trainer Settings->Slope Limit = 30
Settings->Trainer Settings->Advanced Trainer Settings->Slope Scaling Uphill = 80
Settings->Trainer Settings->Advanced Trainer Settings->Slope Scaling Downhill = 150
Settings->Trainer Settings->Advanced Trainer Settings->Wheel Circumference = 2096
Settings->Trainer Settings->Advanced Trainer Settings->ERG Mode reaction time = 5
Settings->Trainer Settings->Speed Adjustments->Power = 180
Settings->Trainer Settings->Speed Adjustments->Speed = 100

What are your Fulgaz settings? Any recommended Fulgaz settings?

**Comment by cagnulein** [comment]
hi @robtheviking in the zip file i don't see the most important file: the debug log :)
can you send me it?

also for the fulgaz settings: i never tried fulgaz so I don't know, but other users already tried it. for example #702 #132 
you can try to comment to the tickets tagging the original poster

Let me know

**Comment by robtheviking** [comment]
Hi Roberto, I enabled the debug log in the settings and quit the app, re-opened it and ran the test but nothing showed up in my iOS (16.4) Files app. That being said...retrying Fulgaz with QZ looks like it works actually. I had to connect to QZ in a specific order and give it a lot of time to connect and maybe it's because the software was updated since I filed the ticket. Anyway, I'll look up another guide on how to get the debug log to work again if I need it in the future, BTW, your website breaks when you try to search in Firefox. It just freezes and you cannot scroll anymore. 
1) https://robertoviola.cloud/qdomyos-zwift-guide/
2) CTRL+F, type anything, the webpage cant scroll anymore.
